itest/opstackl2/op_e2e_test.go

```
@@ -18,32 +21,53 @@ func TestOpSubmitFinalitySignature(t *testing.T) {
 	ctm := StartOpL2ConsumerManager(t)
 	defer ctm.Stop(t)
 
-	// register FP in Babylon Chain
-	fpList := ctm.StartFinalityProvider(t, 1)
+	// start Babylon chain FP
+	_ = ctm.StartFinalityProvider(t, e2etest.ChainID, 1)
```

why do we need to use `e2etest.ChainID`? does any logic depend on this specific value? if not, we should just use a mock value

---

also let's remove `_=`

---

> why do we need to use `e2etest.ChainID`? does any logic depend on this specific value? if not, we should just use a mock value

Yes, we need to use it, while using a mock value, it throws the error:
```
Error:          Received unexpected error:
                the finality-provider manager has already stopped
```